extensions/markdown-language-features/media/markdown.css

```diff
@@ -132,6 +132,9 @@ p {
 }
 
 ul,
+ul{
```

Comment:
Should all unordered list have no bottom margin, or only lists inside of other lists?

Reply:
Hii@mjbvz what do you suggest should be appropriate?

Reply:
I'd keep the existing rule and add new rules  the zero the bottom margin for `ul ul {` (and probably `ul ol`, `ol ul`, `ol, ol` too, although please test those out to see what they look like first)

Reply:
@mjbvz Ok, I will do the same, thanks!